test/index.js

```diff
@@ -450,44 +450,241 @@ describe('locale-specific pluralization rules', function () {
       expect(polyglot.t('n_votes', c)).to.equal(c + ' komentarjev');
     });
   });
+});
 
-  it('pluralizes in Turkish', function () {
```

Comment:
can these tests be kept? deleting tests makes it harder to know if it's a breaking change or not.

Reply:
These are breaking. I deleted these tests because the rules for turkish and lithuanian changed completely, so I opted to cover those cases with new tests checking the exemplar plural rules. Testing each locale's _categorization_ is different from what these tests are doing.